UserGuide.adoc: Edit user guide (#132)
CS2103-AY1819S1-W14-1/main: docs/UserGuide.adoc

By: `W14-1`      Since: `Sept 2018`      Licence: `MIT`

= About this guide

> This guide provides an overview of ClinicIO and shows you how to install ClinicIO.

Reviewer: ```suggestion
This guide provides an overview of ClinicIO and walks you through the installation process.
```

> In addition, this guide provides an in-depth documentation for all the features.

Reviewer: ```suggestion
In addition, this guide provides in-depth documentation for all features.
```

This guide does not assume any prior technical knowledge. Check the <<FAQ>> section for frequent user issues.

== Introduction

> ClinicIO is a desktop app to help you manage clinic with ease, efficient and simplicity.

Reviewer: ```suggestion
ClinicIO is a desktop app to enable you to manage a clinic with ease and efficiency.
```

> You can use ClinicIO to manage patient record, manage the clinic queue, manage clinic's medical inventory, manage clinic appointment and provide statistics on ClinicIO. +

Reviewer: ```suggestion
You can use ClinicIO to manage patient records, queues, medical inventory, appointments as well as view detailed statistics. +
```

Go to the <<Installation Guide>> to get started on install ClinicIO.

== Installation Guide

This section is designed to help you to setup ClinicIO with 3 simple steps.

[NOTE]

> You can move on to <<User Interface Walkthrough>> if you have install ClinicIO.

Reviewer: ```suggestion
You can move on to <<User Interface Walkthrough>> if you have installed ClinicIO.
```
